theme_monglia/views/customizations.xml

```diff
@@ -33,7 +33,7 @@
  <template id="s_cover" inherit_id="website.s_cover" name="Monglia s_cover">
```

Comment:
Primary buttons work better here ;)

| this PR | master |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/f46c9fbe-97ef-4da4-b101-0d44322ecef2) | ![image](https://github.com/user-attachments/assets/616a5320-ee1a-45a0-91aa-63ed527ee2b4) |